Correction: declaring variable of offline sync table

onnkeat opened the pull request:
Correction on code of declaring MobileServiceSyncTable<ToDoItem> variable.
Original is MobileServiceTable<ToDoItem>, the correct one should be MobileServiceSyncTable<ToDoItem> .

azurecla commented:
Hi __@lodipalm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@JennieHubbard 
 Jennie, please review this PR and let me know if I should merge. Thanks.

rmca14 commented:
@barbkess 
FYI

JennieHubbard commented:
Approved!

onnkeat committed:
Update app-service-mobile-android-how-to-use-client-library.md

Correction on code of declaring MobileServiceSyncTable<ToDoItem> variable.
Original is MobileServiceTable<ToDoItem>, the correct one should be MobileServiceSyncTable<ToDoItem> .
